### PR TITLE
Issue/47/save results in usable form

### PR DESCRIPTION
**Pull request by leckijakub**

### CODE REVIEW

File: inference_engine/answers.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"index": "123", "answers": {"1": [1, 0, 0, 0], "2": [0, 1, 0, 0], "3": [0, 0, 1, 0], "4": [0, 0, 0, 1], "5": [0, 0, 0, 0], "6": [0, 0, 0, 0], "7": [0, 0, 0, 0], "8": [0, 0, 0, 0], "9": [0, 0, 0, 0], "10": [0, 1, 0, 0], "11": [0, 0, 0, 0], "12": [0, 0, 0, 0], "13": [0, 0, 0, 1], "14": [0, 0, 0, 0], "15": [0, 0, 0, 0], "16": [0, 0, 0, 0], "17": [0, 0, 0, 0], "18": [0, 0, 0, 0], "19": [0, 0, 0, 0], "20": [0, 0, 0, 0], "21": [0, 0, 0, 0], "22": [0, 0, 0, 0], "23": [0, 0, 0, 0], "24": [0, 0, 0, 0], "25": [0, 0, 0, 0], "26": [0, 0, 0, 0], "27": [0, 0, 0, 0], "28": [0, 0, 0, 0], "29": [0, 0, 0, 0], "30": [0, 0, 0, 0], "31": [0, 0, 0, 0], "32": [0, 0, 0, 0], "33": [0, 0, 0, 0], "34": [0, 0, 0, 0], "35": [0, 0, 0, 0], "36": [0, 0, 0, 0], "37": [0, 0, 0, 0], "38": [0, 0, 0, 0], "39": [0, 0, 0, 0], "40": [0, 0, 0, 0]}}
```
[the remark this answers is not in the document]
Comment (leckijakub) [reply inference_engine/answers.json]: removed

**Comment by leckijakub** [comment]
> There are to much functionality taken from controller. I think exam.py should be split into postprocess functions, like _create_output_json or _get_readable_result, and functions that could be moved to controller, that communicate with other modules.

main function for checking pdf and image moved back to controller.
exam.py and storage.py functions splitted into exam_storage package which is divided further to:
```
├── local_storage.py
├── metadata.py
├── pdf.py
├── remote_storage.py
└── versioning.py
```

File: inference_engine/src/exam_storage/pdf.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import enum
+
+
+class PDFType(enum.Enum):
+    answer_sheets = 1
+    answer_keys = 2
```
Comment (marekhering) [inline inference_engine/src/exam_storage/pdf.py:6]: rename file to pdf_type.py

**Review by marekhering — APPROVED** [review]

File: inference_engine/src/exam_storage/pdf.py (hunk 2 above)
Comment (leckijakub) [reply inference_engine/src/exam_storage/pdf.py]: done

**Review by marekhering — APPROVED** [review]

**Pull request merged**